### DISCUSSION TITLE
About Final Storage and Plots Lifespan

### DISCUSSION THREAD

**Discussion by WMark77** [Chia-Network/chia-blockchain#6116] [General]
Hi,

I intend to fill my 20TB final storage with k-32 plots. I'd apreciate if a good soul could answer those few questions: 

1) Does it possible keep that farming without creating more plots (because I don't have any more final storage capacity)?

2) Do that plots (in my final storage) have a lifespan? Do that plots can be farming for a lifetime or there will be a moment those plots should be replaced by new ones?

Thank you.

**Comment by firmicus** [comment]
1. - yes
2. - https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#plotting  (GoTo: Do my plots "wear out" or "go bad"/"go stale" over time? Will I have to regularly re-plot?)

**Reply by WMark77** [reply]
Thank you.
